grammars/meson.json

```diff
@@ -16,11 +16,11 @@
       "include": "#string_quoted_single"
     },
     {
-      "match": "\\b(if|else|elif|endif|foreach|endforeach)\\b",
+      "match": "\\b(elif|else|if|endif|foreach|endforeach|continue|break)\\b",
       "name": "keyword.control.flow.meson"
     },
     {
-      "match": "\\b(and|not|or)\\b",
+      "match": "\\b(and|not|or|in)\\b",
       "name": "keyword.operator.logical.meson"
```

Comment:
@TingPing Maybe you would want to rename this. (as `in` is not a logical operator)

Reply:
The names aren't defined by me, they are part of themes. Feel free to find another more accurate one.

Reply:
After review this is the correct identifier.